### ISSUE TITLE
webbrowser.get() can hang and should be timeouted

### ISSUE THREAD

**Issue by dg-nvm** [Keeper-Security/Commander#1259]
https://github.com/Keeper-Security/Commander/blob/fe9c026e600f6d60cc636e30ae1546436df18d03/keepercommander/auth/console_ui.py#L264C18-L264C28

webbrowser.get() under some circumstances (my case is WSL2 and wslu from Flox) can hang indefinetelly on xdg-settings:

`xdg-settings get default-web-browser` will just not return anything

This will render UI blank until interrupted with "ctrl+c", then normal keeper UI proceeds

There are multiple issues (e.g https://github.com/microsoft/WSL/issues/8892) touching the subject on the topic but I believe timeouting it will be easiest as there is an Exception catcher there anyway.

**Comment by aaunario-keeper** [comment]
Thanks for shedding light on this issue. Upgrading to Commander v16.11.3 or higher should fix it.

**Comment by dg-nvm** [comment]
I don't think this will fix it as https://github.com/Keeper-Security/Commander/commit/6d15bdbf3e44460c78f4da36faddaae0be27be8a as you will never go out from line 264

**Comment by aaunario-keeper** [comment]
From my own testing, `webbrowser.get()` does not hang when `xdg-open` is not configured properly (i.e., is using the default WSL xdg settings), but does hang when `wb.open_new_tab(url)` is called. This change is intended to prevent the latter from happening if a mis-configuration is detected.
If you have stack-trace evidence contrary to that, I'd be more than happy to see it.

**Comment by dg-nvm** [comment]
My patch:

![image](https://github.com/Keeper-Security/Commander/assets/50369002/62cb87e8-b736-421c-9a37-9aea2868e743)

When it breaks:

![image](https://github.com/Keeper-Security/Commander/assets/50369002/97e9c21e-ebf6-4104-b48d-25945e96754f)

https://github.com/Keeper-Security/Commander/assets/50369002/fc7c33fb-cc83-4d32-9bad-8163f555289c

**Comment by aaunario-keeper** [comment]
I'm not entirely sure what that script is supposed to show. 
Yes, calling `webbrowser.get()` does result in a call to `xdg-settings get default-web-browser`, but when no suitable browser is found, it returns nothing but does not hang (which is what I'm assuming your timeout replacement for the `xdg-settings` call is meant to simulate).
Are you seeing something different? What do you get if you call `xdg-settings get default-web-browser` under the scenario described above?

**Comment by dg-nvm** [comment]
@aaunario-keeper exactly, there are problems with dbus on WSL and `dbus-send` that xdg-settings bash script uses to detect if perhaps browser is set in Gnome Settings manager is hanging - 

https://github.com/alyptik/dotfiles/blob/2c3d5a5c9886e998042ea4eee21097241a65a890/xdg-settings#L465

for testing purpose I replaced whole script with sleep but it's hanging on the line I pasted here
